### PR TITLE
Documentation ISLE local install corrections

### PR DESCRIPTION
**Pull request by dwk2**
Documentation: ISLE local install corrections
- Added Windows instructions: Enable Docker Shared Drives
- ISLE local install: Removed duplicate text for Mac and Windows installs (etc/hosts file) 
- Fixed urls that were still pointing to "ISLE-Documentation" repo or github.io pages
- Fixed Windows instructions to install mkdocs theme

### CODE REVIEW

File: docs/install/host-software-dependencies.md [hunk 1]
```diff
@@ -309,7 +307,12 @@ git version 2.15.1
 
 * One can see a whale icon in the notification area.  
 
-* Please note: This process also installs the newest version of `docker-compose`.
```
Comment (bseeger) [inline docs/install/host-software-dependencies.md]: Is this not true or unimportant?

Comment (dwk2) [reply docs/install/host-software-dependencies.md]: I appreciate you questioning that removal. My thinking was that this added confusion or "extra unnecessary information" for the Mac & Windows installation process as that process is for newbies or local dev environments (or both) and we don't mention in that process that `docker-compose` is required. Yes, I was streamlining. However, after re-reading the Ubuntu and CentOS installations, I think I am in agreement w/ you that it makes more sense to keep it. 

How about this idea? Keep it and call out it's importance by adding in a stub for installing Docker-Compose as is done w/ Ubuntu and CentOs? How's that land for you? 
For example, below has a new "Step 3" inserted to show that:

### Step 2: Install Docker for Mac
...
### Step 3: Install Docker-Compose
- The Docker installation already installed the newest version of `docker-compose`.
### Step 4: Clone ISLE Repository
...

Comment (bseeger) [reply docs/install/host-software-dependencies.md]: This sounds like a good idea.  I didn't really have an opinion on keeping it or not, just wanted to make sure that someone reading the whole page doesn't get confused as to why docker-compose isn't mentioned for a few systems.  I like what you've done with the Steps above - I think it's a good idea to mention it and that it was already installed.

Comment (dwk2) [reply docs/install/host-software-dependencies.md]: Added text as indicated above to clarify installation of `docker-compose` on Win and Mac.
Also removed a stray "`" mark from one of the CentOS copy and paste commands.

**Pull request merged**